wifiphisher/common/interfaces.py

```diff
@@ -243,6 +243,36 @@ def up_ifaces(self, ifaces):
         for i in ifaces:
             card = pyw.getcard(i.get_name())
             pyw.up(card)
+    
+    def _generate_random_address(self):
```

Comment:
I think this should be a function instead of a method in the `interfaces` module since it only returns a string and doesn't have to do anything with the object. A `set_interface_mac` should be implemented in the `NetworkManager` object to change the mac address.